### DISCUSSION TITLE
Embedded vs. Attached Image

### DISCUSSION THREAD

**Discussion by johnfitz00** [zbateson/mail-mime-parser#195] [General]
Any ideas how to determine if an attachment is an embedded image, or an attached one?

**Comment by zbateson** [comment]
@johnfitz00 easiest would be to use https://mail-mime-parser.org/api/2.2/classes/ZBateson-MailMimeParser-Message-IMessagePart.html#method_getContentDisposition

You could call it with a default depending on what you would want to consider 'null'... e.g.

``` $myImagePart->getContentDisposition('inline'); ```

All the best
